src/include/ipc4/base_fw.h

```diff
@@ -667,3 +667,8 @@ struct ipc4_log_state_info {
 	 */
 	uint32_t logs_mask[IPC4_MAX_LIBS_COUNT];
 } __attribute__((packed, aligned(4)));
+
+enum ipc4_power_state_type {
+	ACTIVE_CORES_MASK = 0,
```

Comment:
can we prefix to avoid any naming collisions/confusion ?

Reply:
Sure
Also:
File name changed.
I was thinking about having a pause/resume mechanism for overriding the managers clock choice, but according to @mmaka1 it is currently sufficient. This can be easily worked around by requesting artificial kcps budget. We were also considering an approach of holding a reference of who requested the clock and decide on the change on clock driver level, but left that topic for later.